### PR TITLE
Add sort command

### PR DESCRIPTION
**Pull request by xiaoying1129**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -168,4 +169,26 @@ public String toString() {
...
+     */
+    public int compareTo(Person p) {
+        requireNonNull(p);
+        Double curr = this.getSubjectsTaken()
+                .stream()
+                .map(Subject::getTotalPercentage).mapToDouble(Double::doubleValue).sum();
+        Double comp = p.getSubjectsTaken()
+                .stream()
+                .map(Subject::getTotalPercentage).mapToDouble(Double::doubleValue).sum();
+        return curr.compareTo(comp);
```
Comment (e1010101) [inline src/main/java/seedu/address/model/person/Person.java:196]: Pretty good!

File: src/main/java/seedu/address/model/person/UniquePersonList.java [hunk 2]
```diff
@@ -134,4 +134,14 @@ private boolean personsAreUnique(List<Person> persons) {
...
+
+    /**
+     * Utilizes Person's compareTo() function provide a comparator for ObservableList<>.sort
+     * @param isInAscending sorts by ascending if true, descending otherwise
+     */
+    public void sortByDate(boolean isInAscending) {
+        internalList.sort((Person p1, Person p2) -> isInAscending
+                ? p1.compareTo(p2)
+                : -p1.compareTo(p2));
+    }
```
Comment (e1010101) [inline src/main/java/seedu/address/model/person/UniquePersonList.java]: How is this comparing by date?

**Review by e1010101 — CHANGES_REQUESTED** [review]
Please fix checkstyle + add relevant javadocs to public methods! Otherwise, LGTM!

**Review by e1010101 — APPROVED** [review]
ok!

**Review by jgyj123 — APPROVED** [review]
LGTM :+1

**Pull request merged**